deployments/jwebbinar/image/environments/post-start-hook

```diff
@@ -8,4 +8,16 @@ set -eu
 # be performed both on AWS and standalone.  AWS-only functions should be included
 # directly in common.yaml or perhaps another script.
 
+rsync -r /etc/default-home-contents/ /home/jovyan
+
 /opt/environments/setup-notebooks
+
+chmod +x /home/jovyan/Desktop/*.desktop
```

Comment:
I'm not sure that we need this.  What files fit the `*.desktop` pattern?

Reply:
ok, will delete this line